apps/src/code-studio/pd/form_components_func/FormController.jsx

```diff
@@ -119,6 +119,7 @@ const FormController = props => {
   useEffect(() => {
     onInitialize();
     onSetPageInternal(currentPage);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
```

Comment:
```suggestion
  }, [onInitialize, onSetPageInternal, initialPage]);
```

Reply:
and then up on line 85:
```
  // We use functions here as the initial value so that these values are only calculated once
  const initialPage = useMemo(() => getInitialStored(sessionStorageKey, 'currentPage') || 0, [sessionStorageKey])
  const [currentPage, setCurrentPage] = useState(initialPage);
```

Reply:
I get `React Hook useEffect has a missing dependency: 'currentPage'` is it ok to add currentPage?

Reply:
Could we either (a) disable the linting rule for this so that it's truly happening on mount, or (b) remove/modify the comment above this hook?

Reply:
oh sorry, also to replace currentPage with initialPage on line 121.
This is intended to run on mount so it should only ever be called when currentPage is equal to initialPage